README.md

```diff
@@ -33,10 +33,10 @@
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [Awesome books] <a name="about-project"></a>
+# 📖 [LeaderBoard] <a name="about-project"></a>
 
 
-**WebPack and Linters Template** This is a template Repository for webpack project with Linters pre-installed
+**LeaderBoard** The leaderboard website displays scores submitted by different players. It also allows you to submit your score. All data is preserved thanks to the external Leaderboard API service.
 
```

Comment:
- [ ] Live demo link one _line 64_ is broken, kindly fix it!

Reply:
@roodjinocherilus 
We ought to remove the link in markdown, as it remained clickable in the ReadMe preview.

Cheers! 😃

Reply:
Needs attention!